examples/generative/gpt2_text_generation_with_kerasnlp.py

```diff
@@ -25,16 +25,23 @@
 """
 
 """
-## Install KerasNLP and Import Dependencies
+## Install KerasNLP, Choose Backend and Import Dependencies
+
+We can choose one of "tensorflow", "jax", "torch" as the backend. Let's go
+ahead with JAX in this example.
 """
 
 """shell
 pip install -q keras-nlp
 """
 
+import os
+
+os.environ["KERAS_BACKEND"] = "jax"
```

Comment:
Leave a comment maybe `os.environ["KERAS_BACKEND"] = "jax"  # or "tensorflow" or "torch"`

Reply:
Also, still not sure if we would want this to default to `"jax"` or `"tensorflow"`. We should figure out how we want to approach generally.